react/tic-tac-toe/src/app/screens/Login/components/fields/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const getValidityClassName = meta => {
+  if (meta.active) {
+    return '';
+  }
+  if (meta.touched && meta.invalid) {
+    return " invalid";
+  }
+  if (meta.touched && meta.valid) {
```

Comment:
This conditions could be checked inline inside the component function like this
```js
const inputClassName = `meta.touched ? meta.valid ? styles.valid : styles.invalid : '';`
```
Also, remember to use css modules instead of the class string.